pages/api/report-bot.ts

```diff
@@ -0,0 +1,160 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { Octokit } from '@octokit/rest'
+import { withSessionRoute } from '@/lib/session'
```

Comment:
I think this line is the last remaining build error, see my comment in the PR. (3x, because it's in 3 files.)